### ISSUE TITLE
Some questions about articles and code

### ISSUE THREAD

**Issue by booker-max** [AlbertoSabater/EventTransformer#1]
Hello, this work is very wonderful and gave me a lot of inspiration, but I don't understand it very well in some places, especially the input parameters of the network
1.  For the event of a video clip, you store it in a matrix of size **(t_during/chunk_len_us)*H*W*2**, for the DVS128 dataset, chunk_len_us = 12000us, according to the paper **one for each polarity pϵ{0 , 1}**, but when you save it as a sparse matrix, it does not show that positive or negative is considered. The specific code is as follows:
![image](https://user-images.githubusercontent.com/74257220/170293118-c36e80ff-08e0-4a0f-999e-59f31761cee0.png)

2.  In DVS128, you set bins = 3, In SL-Animals-DVS, you set bins =2, I know these are hyperparameters, I want to ask what is the relationship between **bins** and **chunk_len_ms**, chunk_len_ms is the time_window_lengths ∆t in the article, actually I don't know the function of introducing **bins** and **time_window_lengths** parameters

3. In your data augmentation, you set "max_sample_len_ms = 504ms", because I will do experiments on a new dataset later, I would like to know how to set this "max_sample_len_ms"?

Looking forward to hearing from you, thank you!

**Comment by booker-max** [comment]
Sorry to bother you, for the first question, I have solved.
But for the second and third question, I still have questions.

**Comment by AlbertoSabater** [comment]
Hello! Thank you for the comment and sorry for the late response:
2. 
__chunk_len_ms__ (time_window_length) stands for the length (in ms) of the frame-like representation. The longer it is the more information it captures. 
__bins__ is used to group the information within the frame-like representation, the more bins the more detailed the representation will be, but also bigger. 
Both values are selected depending on the evaluation datasets. They affect the final accuracy but also the efficiency (Table 5).

3. __max_sample_len_ms__ depends also on the event recordings. Some of the recordings used in my work are shorter so the max_sample_len_ms must be shorter. Also, bigger ones increase the computational requirements during training.
